system/web/services/InterceptorService.cfc

```diff
@@ -288,6 +290,23 @@ component extends="coldbox.system.web.services.BaseService" accessors="true" {
 		);
 	}
 
+	/**
+	 * Allow point-first arguments when calling listen() or unlisten().
+	 *
+	 * Basically flips the `target` and `point` arguments if the former is found to be a string instead of closure.
+	 *
+	 * @target Could be the target closure... could be the listen point.
+	 * @point  Could be the interception point... could be the target closure
+	 */
+	private struct function normalizeListenArguments( required target, required point ){
+		if ( isValid( "string", arguments.target ) ) {
```

Comment:
I would probablly test if `isSimpleValue()` is faster than `isValid()`

Reply:
You're 100% right, Luis. isSimpleValue is slightly faster in Lucee and much, much faster in ACF 2021.

https://www.trycf.com/scratch-pad/gist/c98a8a7eda138606c060883f7f9b372f

Reply:
Done. :+1: